pages/management/ManagementHandler.inc.php

```diff
@@ -401,25 +405,25 @@ public function access($args, $request)
         $apiUrl = $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), 'contexts/' . $context->getId());
         $notifyUrl = $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), '_email');
         $progressUrl = $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), '_email/{queueId}');
-        $userGroups = DAORegistry::getDAO('UserGroupDAO')->getByContextId($context->getId());
 
         $userAccessForm = new \APP\components\forms\context\UserAccessForm($apiUrl, $context);
-        $notifyUsersForm = new \PKP\components\forms\context\PKPNotifyUsersForm($notifyUrl, $context, $userGroups);
+        $enableBulkEmails = in_array($context->getId(), (array) $request->getSite()->getData('enableBulkEmails'));
+        $notifyUsersForm = $enableBulkEmails ? new PKPNotifyUsersForm($notifyUrl, $context) : null;
```

Comment:
In Smarty there's an "{if}" checking the `$enableBulkEmails`, if it's false, the `PKPNotifyUsersForm` isn't used, so I also moved the `$userGroups` inside it, to avoid issuing a query.

Reply:
ps: I didn't add this improvement to the stable-3_3_0, I think I'll do it (in case these forms exist there).